src/lib/context/ContextWrapper.tsx

```diff
@@ -60,8 +60,8 @@ const ContextWrapper: FC<{ children: ReactElement }> = ({ children }) => {
                     >
                       WebXDAO/DevConf
                     </a>
-                    <br /> 2. Add Event data in markdown file
-                    <br /> 3. Create pull request and Your event will be live.
+                    <br /> 2. Add event detail in the issue.
```

Comment:
Can you please use block elements instead of using line breaks?

Reply:
Hey, i'm ok with the suggestion. Let's accept the changes and optimize later?